### DISCUSSION TITLE
Submitting google models to leaderboard without setting up billing accounts on vertexai

### DISCUSSION THREAD

**Discussion by akshaykripalani** [embeddings-benchmark/mteb#1615] [Q&A]
I want to submit the performance of google's text-embedding-005 model to the mteb leaderboard. I've read through the guide on how to add a model, contribution guidelines, but i cannot figure out how i can run this benchmark without setting up a billing account for vertexai.

Included is the code i wrote 
```
import mteb
from mteb.models.google_models import google_text_emb_005

model = google_text_emb_005.loader()

tasks = mteb.get_benchmark("MTEB(eng, classic)") 

evaluation = mteb.MTEB(tasks=tasks)
evaluation.run(model, output_folder="results")

```

Is there a way to do this using API keys? When I am developing with embedding models it lets me generate embeddings using that.

**Comment by Samoed** [comment]
What API keys do you want to use?

**Reply by akshaykripalani** [reply]
Using API Keys from Google AI Studio gives you access to Generative Language API, which lets you access models like text-embedding and gemini in code without having to set up billing. This is different from Vertex AI API, which requires you to set up a billing account before you can proceed with it.
